pettingzoo/utils/wrappers/base_parallel.py

```diff
@@ -17,37 +22,47 @@ def __init__(self, env):
 
         # Not every environment has the .state_space attribute implemented
         try:
-            self.state_space = self.env.state_space
+            self.state_space = self.env.state_space  # type: ignore
         except AttributeError:
             pass
 
-    def reset(self, seed=None, options=None):
+    def reset(
+        self, seed: int | None = None, options: dict | None = None
+    ) -> tuple[Any, Any]:
```

Comment:
I think we can narrow down the return type here?

Reply:
Changed reset to return `tuple[ObsDict, dict[str, dict]]`